### DISCUSSION TITLE
How to phrase docker run -e ?

### DISCUSSION THREAD

**Discussion by sk0003mk** [wg-easy/wg-easy#544] [Q&A] [answered]
Can anyone give me an example of how to phrase the command docker run -e to set different allowed IPs? The container name is wgeasy. So convoluted, no clear instructions anywhere.

**Comment by eandersons** [comment]
That would be `-e WG_ALLOWED_IPS='192.168.15.0/24, 10.0.1.0/24'` (example taken from [Options](/WeeJeWel/wg-easy#options)).

**Reply by sk0003mk** [reply]
Hello, thank you for the response. But again this does not work. It says docker run requires at least 1 argument. How would you do the full command? docker run in front of this got me this result.

**Reply by eandersons** [reply]
Then you should post the full `docker run` command to see why it causes the mentioned error.
My guess is that either there is some quoting related issue or you are trying to run incomplete `docker run` command.

Is a WireGuard Easy container already running and you want to change value of the environment variable `WG_ALLOWED_IPS`?
If so, then you have to remove the container (`docker rm wgeasy`) and recreate by adjusting the `docker run` command from [Run WireGuard Easy](/WeeJeWel/wg-easy#2-run-wireguard-easy) to fit your needs.

**Reply by sk0003mk** [reply]
Yes, I already have a WireGuard Easy container running and I want to modify the value of the environment variable. What do I need to do in that case? Why do I need to remove the container?

**Reply by eandersons** [reply, marked as answer]
As far as I know, it is not possible to change environment variables for a running process in a Docker container, therefore you need to recreate it (remove and run it again).
